eks-ebs-enable/README.md

```diff
@@ -0,0 +1,169 @@
+# EBS CSI Driver Installer
+
+> NOTE: only use this tool as directed by SUSE support. Any usage outside of this is unsupported.
+
+## Purpose
+
+Starting with EKS v1.23 you are required to use the out-of-tree drivers for EBS backed volumes. This support tool can be used to enable the the **Amazon EBS CSI Driver** via an EKS addon.
+
+When run this will do the following per cluster:
+
+- Lookup the cluster details in Rancher
+- Get the AWS creds for the cluster from Rancher (if not using explicit creds)
+- Get the EKS cluster details from AWS
+- Creates an OIDC provider for the EKS cluster
+- Create a new IAM role for the cluster to use with the EBS addon (named `EBS_CSI_<CLUSTERNAME>` )
+- Attaches the EBS policy to the new role
+- Installs the EBS CSI addon for the EKS cluster using the created role
+- Waits for the addon to be `active` (or timesout)
+
+The tool is idempotent so it can be run multiple times against the same cluster (even if the last run failed).
+
+> NOTE: this tool only runs against standard AWS regions (and not US GovCloud or China regions)
+
+## Requirements
+
+You will need specific permissions to enable the addon. These permissions are currently outside the recommended set of permissions for Rancher.
+
+There are 2 options to supply these permissions
+
+### Option 1 - augment the permissions used when provisioning the cluster
+
+The IAM user who's credentials where used to provision the cluster can be augumented by adding the following permissions (via a policy). To do this with the `aws` cli you can run the following command to create the policy:
+
+```shell
+aws iam create-policy --policy-name Rancher_Support_EKSEBS --policy-document `
+`{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "eks:DescribeAddonConfiguration",
+                "eks:UpdateAddon",
+                "eks:ListAddons",
+                "iam:CreateOpenIDConnectProvider",
+                "eks:DescribeAddon",
+                "iam:ListOpenIDConnectProviders",
+                "eks:DescribeIdentityProviderConfig",
+                "eks:DeleteAddon",
+                "eks:CreateAddon",
+                "eks:DescribeAddonVersions",
+                "sts:AssumeRoleWithWebIdentity",
+                "eks:AssociateIdentityProviderConfig",
+                "eks:ListIdentityProviderConfigs"
+            ],
+            "Resource": "*"
```

Comment:
```suggestion
            "Resource": [
                "*"
                ]
```

Reply:
Not changed, the current version is less verbose